app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -130,7 +130,7 @@ def copy_state_id_address_to_residential_address(pii_from_user)
       end
 
       def updating_state_id?
-        pii_from_user.has_key?(:first_name)
+        pii_from_user.has_key?(:first_name) && !user_session[:idv][:ssn].nil?
```

Comment:
Can we add some test coverage for this change?

Also, is `user_session[:idv]` guaranteed to exist so that we don't run into an error `undefined method `[]' for nil`, or should we use `dig` instead?

```suggestion
        pii_from_user.has_key?(:first_name) && user_session.dig(:idv, :ssn).present?
```

Reply:
i made this update and also added test coverage